test/strategy/anchor/AnchorUSTStrategy.spec.ts

```diff
@@ -521,6 +521,32 @@ describe("AnchorUSTStrategy", () => {
 
       expect(await strategy.redeemOperationLength()).equal(2);
     });
+
+    it("Returns array of depositOperations when calling getter function", async () => {
```

Comment:
These cases are not related with `#initRedeemStable` function.
Can you make new `describe`?

Reply:
makes sense!

I will kill the PR because of discussion had on slack

Next time I add tests I'll make sure that the `describe` block that they're in makes sense in terms of what the `it` cases do

thanks!